metricbeat/tests/system/metricbeat.py

```diff
@@ -19,8 +19,12 @@ class BaseTest(TestCase):
 
     @classmethod
     def setUpClass(self):
-        self.beat_name = "metricbeat"
-        self.beat_path = os.path.abspath(os.path.join(os.path.dirname(__file__), "../../"))
+        if not hasattr(self, 'beat_name'):
+            self.beat_name = "metricbeat"
+
+        if not hasattr(self, 'beat_path'):
+            self.beat_path = os.path.abspath(os.path.join(os.path.dirname(__file__), "../../"))
+
```

Comment:
Since Auditbeat with X-Pack is in its own `xpack/auditbeat` directory, `beat_path` needs to point to that. It was being overwritten since `AuditbeatXPackTest` in `auditbeat_xpack.py` ultimately extends `BaseTest` from `metricbeat.py`.